packages/tools/client-debugger/client-debugger-view/src/components/FluidDataView.tsx

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+import { HasContainerId, VisualNode, VisualNodeKind } from "@fluid-tools/client-debugger";
+import { FluidHandleView } from "./FluidHandleView";
+import { TreeView } from "./TreeView";
+import { ValueView } from "./ValueView";
+import { UnknownDataView } from "./UnknownDataView";
+
+/**
+ * {@link FluidDataView} input props
+ */
+export interface FluidDataViewProps extends HasContainerId {
+	containerId: string;
+	node: VisualNode;
+}
+
+/**
+ * Displays visual summary trees for DDS_s within the container
+ */
+export function FluidDataView(props: FluidDataViewProps): React.ReactElement {
+	const { containerId, node } = props;
+
+	let view: React.ReactElement;
+	switch (node.nodeKind) {
+		/**
+		 * node with children
+		 * TreeNodeBase
+		 */
+		case VisualNodeKind.TreeNode:
+		case VisualNodeKind.FluidTreeNode:
+			view = <TreeView containerId={containerId} node={node} />;
+			break;
+		/**
+		 * node with primitive value
+		 * ValueNodeBase
+		 */
+		case VisualNodeKind.ValueNode:
+		case VisualNodeKind.FluidValueNode:
+			view = <ValueView containerId={containerId} node={node} />;
+			break;
+		/**
+		 * unknown node type
+		 */
+		case VisualNodeKind.FluidUnknownObjectNode:
+		case VisualNodeKind.UnknownObjectNode:
```

Comment:
We'll want two different components for these 2 unknown data cases, as they are (subtly) different. 

`UnknownObjectNode` indicates we saw data that is not a SharedObject at all (e.g. it was a data store, or some arbitrary application data).

`FluidUnknownObjectNode` indicates we saw a SharedObject, and even have its type and ID, but our system doesn't know how to render it.

We'll want to print two different disclaimers for these cases.